.github/workflows/sonarqube.yml

```
@@ -18,10 +18,33 @@
           fetch-depth: 0  # Important for SonarQube to see history
+
+      - name: Set up Python 3.12
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.12'
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r backend/requirements.txt
+      
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)